extensions/mssql/src/contracts.ts

```diff
@@ -1217,4 +1217,12 @@ export namespace ExecutionPlanComparisonRequest {
 	export const type = new RequestType<ExecutionPlanComparisonParams, azdata.executionPlan.ExecutionPlanComparisonResult, void, void>('queryExecutionPlan/compareExecutionPlanGraph');
 }
 
+export interface IsExecutionPlanXmlParams {
```

Comment:
if we're going to special-case parsing execution plan XML for MSSQL do you think we really need to send a message to STS for this?  it seems like a lot of overhead compared to just parsing the XML in the UI.  If we're going to send messages to the service processes we might as well implement more generically to be a "cell file opener" callback like we had previously discussed, IMO.